### ISSUE TITLE
how to run your code?

### ISSUE THREAD

**Issue by IamMegatron2025** [ecobost/cnn4brca#1]
hello, I am interest at your code. But how to run your code?  can you give some advice？
beside, can you provide some trainning data?

**Comment by ecobost** [comment]
I added some code on how to run in the README.md: https://github.com/ecobost/cnn4brca

**Comment by IamMegatron2025** [comment]
@ecobost thank your response. can you provide a good pre-trained model?
thank you very much!

**Comment by Maya1994** [comment]
@IvyGongoogle did u get the database ? i asked them but they didn't reply !

**Comment by IamMegatron2025** [comment]
@Maya1994 sorry, I have not got it

**Comment by Maya1994** [comment]
did u try the code with other dataBase ? can we try it with the MIAS data base ??

**Comment by ecobost** [comment]
@Maya1994 Hi, people at BCDR do take some time to answer, I have not tried to run this code in a different database but it should be possible although with some modifications

**Comment by Maya1994** [comment]
@ecobost  here is an example of mammograms that I have, as I told you i'm interested with microcalcification detection, 
![image11](https://cloud.githubusercontent.com/assets/27205506/25902459/7c32b910-3591-11e7-9f96-64d4409cf51d.jpg)
![image11_label](https://cloud.githubusercontent.com/assets/27205506/25902470/811a2418-3591-11e7-9a5a-646b028ac8a8.jpg)

You asked me before to generate masks first , but how can I do it ? as u said in ur previous comment  : " what createMask.m does is take an outline (a series of x,y coordinates) and creates the contour of the lesion from it" but i don't have this x and  y !!! 

Many Thanks,

**Comment by ecobost** [comment]
Do you have any ground truth of where the lesions are? that has to be provided to the network as the label for training otherwise it will have no examples to learn from. If you do have some ground truth as masks, outlines or any other form, then you would have to change it to the form used by the code (with 0 for background, 127 for normal tissue and 255 for the lesion/positive class). This should probably come with the dataset (and are usually provided by trained radiologists), if you do want to label the images yourself you could try using [labelme](http://labelme.csail.mit.edu/Release3.0/) or [sloth](http://sloth.readthedocs.io/en/latest/index.html) (I haven't used them but they seem relatively easy to use)

**Comment by EngNala** [comment]
mias data base provide x,y image-coordinates of centre of abnormality.

**Comment by shubhamnatani** [comment]
hey are you run the whole code.?
the code gives some error when i run .
please help if you know.?

**Comment by shubhamnatani** [comment]
@ecobost sir the code gives an error. can you help me out.?
